Add '-r' rm file ability

TerraTech opened the pull request:
1. strip-docker-image: Fix bug where multiple '-f' were not appending to $FILES
1. Add -r' option to remove files from final image
1. misc mcedit syntax highlighting fix where it gets confused with multiple quotes in an $() construct

As an aside, can you turn on issues for your fork?  It seems that your fork is the best quality I've seen thus far as the official repo seems to have become neglected.

Also, if you need for me to squash these commits, just let me know.

Commit:
strip-docker-image: Make mcedit happy

Commit:
Add '-r' option to remove files from the final image

Commit:
Misc wording fix to denote '-f' globbing

TerraTech commented:
Note: I chose '-r' over '-d' as I've seen in other forks '-d' was being used to specify reading from a Dockerfile.

Commit:
Add '-r' option to the README.md

lkwg82 commented:
> As an aside, can you turn on issues for your fork?

of course

lkwg82 commented on bin/strip-docker-image-export:
```diff
@@ -77,7 +78,10 @@ function parse_commandline() {
 		    ;;
 		f)
-		    FILES="$FILES $(printf "${OPTARG}")"
+		    FILES+=" $(printf "${OPTARG}")"
```
is this by intention?

TerraTech replied:
Well it was, until you fixed argument handling in: https://github.com/lkwg82/strip-docker-image/commit/fca7cf7faca2488a6496039edaf179ee38109c83

My pull request pre-dated that commit, and I had utilized bash's += operator...

lkwg82 replied:
aah, sry oversaw it.

lkwg82 commented:
plz update the readme.md as well

TerraTech commented:
Given that all this started with fixing '-f', it snowballed into my adding '-r' as well (for a particular project I was currently working on).  Are you interested in adding 'remove' ability?  I also had updated the readme.me file and added '-r' to it and I'm not sure what else is needed to be added.

lkwg82 reviewed (changes requested):
just a test

lkwg82 reviewed (changes requested):
```bash
# strip-docker-image -i ubuntu -t ubuntu-stripped-tar -p tar
/mybin/strip-docker-image-export: line 81: printf: -r: invalid option
printf: usage: printf [-v var] format [arguments]
tar: Removing leading `/' from member names
Sending build context to Docker daemon 3.281 MB
Step 1 : FROM scratch
 ---> 
Step 2 : ADD export /
 ---> fa1999b98b1c
Removing intermediate container 0639cdf49265
Step 3 : CMD /bin/bash
 ---> Running in 2897936655af
 ---> 9c1faca3e887
Removing intermediate container 2897936655af
Successfully built 9c1faca3e887
```
Something went wrong

Commit:
Add '-r file' to SYNOPSIS and usage()

Commit:
strip-docker-image: Fix bug in join()

  Do not perform the join if only SEP is passed

TerraTech commented:
@lkwg82 it is a simple fix to the join() function, however I'm working on rebasing to your latest commits.

Commit:
Merge branch 'add_rm'

TerraTech commented:
@lkwg82 I've rebased and force pushed which fixes the errant join assumption and only contains the new _rm_ ability.  I removed the '-f' fix as your commit resolves that issue: https://github.com/lkwg82/strip-docker-image/commit/fca7cf7faca2488a6496039edaf179ee38109c83
